signals/parser/schema.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, name, json_fields):
             self.create_field(field_name, field_attributes)
 
     def create_field(self, field_name, field_attributes):
-        field_attributes = field_attributes.split(",")
+        field_attributes = [field_attribute.strip() for field_attribute in field_attributes.split(",")]
```

Comment:
I think you could do this more concisely / without the list comprehension if you used regex. Something like: `re.split(", *", field_attributes)